src/scripts/main.js

```diff
@@ -1,3 +1,54 @@
 'use strict';
 
-// write code here
+const buttonAddRow = document.querySelector('.append-row');
+const buttonRemoveRow = document.querySelector('.remove-row');
+const buttonAddColumn = document.querySelector('.append-column');
+const buttonRemoveColumn = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+
+const tableData = {
+  maxRows: 10,
+  minRows: 2,
+  maxColumns: 10,
+  minColumns: 2,
+};
+
+buttonAddRow.addEventListener('click', () => {
+  const newRow = table.insertRow(-1);
+
+  for (let i = 0; i < table.rows[0].cells.length; i++) {
```

Comment:
Consider using a functional programming method like `forEach` instead of a traditional loop to iterate over table rows.